lib/dor/text_extraction/speech_to_text.rb

```diff
@@ -49,6 +64,11 @@ def job_id
         "#{bare_druid}-v#{cocina_object.version}"
       end
 
+      # return the s3 location for the output files generated by the speech to text service
+      def output_location
+        "#{job_id}/output"
+      end
```

Comment:
see https://github.com/sul-dlss/speech-to-text/issues/33#issuecomment-2419960981